tutorial_part1.md

```diff
@@ -1,34 +1,17 @@
-# Title
-...
-This tutorial will be heavily inspired on the ArcBees developer's website [tutorial](TODO).
+# GWTP Beginner's tutorial
+When I first heard about GWT or "gwit", I thought it sounded like some kind of STD, but when I realized it was Java compiling into JavaScript, it got all my attention. Like many people, except the purists (which I respect), I'm not a big fan of JavaScript. Especially when it comes down to writing logic. So here we have GWTP that does the hard work for you and offers you tools that implements all the best practices on how to build web applications with GWT.
 
-...
-You won't be able to send your toaster into space after this tutorial, but it will be close enough...
-...
-
-## Why GWT/GWTP is awesome
-When I first heard about GWT/GWTP I thought it was some kind of STD, but when I realized it was Java compiling into JavaScript, it got all my attention. Like many people, except the purists (which I respect), I'm not a big fan of JavaScript. Especially when it comes down to writing logic. So here we have GWTP that does the hard work for you and offers you tools that implements all the best practices on how to build web applications with GWT.
+This tutorial will be heavily inspired on the ArcBees developer's website [tutorial][tutorial]. You won't be able to send your toaster into space after this tutorial, but it will be close enough...
 
 ## Getting started
-To get started, I strongly suggest watching our how-to video on [how to create a basic project](TODO). This will create a basic "Hello World!" application for you generated from our Maven [archetypes](TODO).
-
-## Creating a basic form
-From the basic archetype, we're going to create what we need in order to send our toaster into space. So, let's create a basic POJO representing our toaster:
-
-```java
-public class FlyingToaster() {
-    private String coordinates; // X, Y, Z separated by semicolons ';'
-    private String power;
+To get started, I strongly suggest watching our how-to video on [how to create a basic project][generate]. This will create a basic "Hello World!" application for you generated from our Maven [archetypes][archetypes].
 
-    ...
-}
-```
+## The Visual: Writing the View
+From the basic archetype, we're going to create what we need in order to send our toaster into space.
 
-In GWTP, there's this concept of Presenter-View pair. They always come together. The Presenter handles the logic and the View only displays what it's told to. **The Presenter don't care about what the View does and the View just don't know anything at all.**
+In GWTP, there's this concept of Presenter-View pair. They always come together. The [Presenter][presenter] handles the logic and the [View][view] only displays what it's told to. So to create our form, we won't need a Presenter yet. What we need is a View class that will use a [UiBinder][uibinder].
```

Comment:
I'd explain what are the roles of Presenter/View.
I'd also explain briefly what the UiBinder is (or what it's used for) as well as providing a link, just like you did.